api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -145,14 +144,40 @@ message Bootstrap {
   Admin admin = 12;
 
   // Optional overload manager configuration.
-  overload.v3.OverloadManager overload_manager = 15;
-
-  // Enable :ref:`stats for event dispatcher <operations_performance>`, defaults to false.
-  // Note that this records a value for each iteration of the event loop on every thread. This
-  // should normally be minimal overhead, but when using
-  // :ref:`statsd <envoy_api_msg_config.metrics.v3.StatsdSink>`, it will send each observed value
-  // over the wire individually because the statsd protocol doesn't have any way to represent a
-  // histogram summary. Be aware that this can be a very large volume of data.
+  overload.v3.OverloadManager overload_manager = 15
+      [(udpa.annotations.security) = {
+        configure_for_untrusted_downstream: true
+        configure_for_untrusted_upstream: true
+        example_configs {
+          example:
+            "  refresh_interval: 0.25s\n"
```

Comment:
Protobuf strings don't support multi-line, so you have this ugliness with "<blah>\n". The alternative is we have a text proto fragment here buried inside an `Any`, but when I tried that, it caused the protobuf compiler to hang, so we'd need to debug that. The other downside of text proto is that it's an unfamiliar syntax to most folks.

Reply:
@lizan @mattklein123 WDYT about the best approach to handling this? I'm almost starting to think that putting examples inline might not be the greatest idea, but in this prototype it does seem to work reasonably well, see the generated docs. From an end user UX, it's helpful to have these directly attached to the fields.

Reply:
I would start with annotations and possibly a comment to sec config docs.  The issue is 1 size does not fit all limits and timeouts when operating with untrusted downstreams and upstreams.  Typical example is H2 flow control windows; setting them too small will result in poor performance when the remote endpoint has high RTT as is typical for a downstream client somewhere in the internets (or a cruise ship).

Reply:
@antoniovicente what do you mean by annotations here? Does this example constitute an annotation from your view point?

Reply:
I was referring to things like configure_for_untrusted_upstream.  Those are great.  I think example config needs to be reconsidered; we can always introduce it later if we decide it is necessary.

Reply:
The another downside of annotation is it baked into all generated code so if we end up example in many places it increases code size as well.

Reply:
Yeah, I think we can drop the experiment here on inline examples due to size and poor proto representation capabilities. I'll switch to the simpler configure_foo annotations.

Reply:
sgtm. I do like the idea of baking in defaults for different "profiles" but maybe we can figure that out separately.

Reply:
I have an alternative idea on how to achieve the programmatic association of examples or defaults with API protos. We could use a similar approach to what I want to do for proxy-specific docs comments. We start with the normal protos as defined today. We then add a manifest file, which is a map from <path to field> to comment or example, e.g.

```
envoy.config.listener.v3.Listener.foo_field: "foo isn't implemented yet in super-duper proxy.
envoy.config.listener.v3.Listener.bar: "bar is alpha in super-duper proxy.
```

Protodoc can then glue this manifest as it generates docs for each proto.

We can do a similar thing for these secure default examples, getting the same end results as in this PR, but without modifying any of the protos, and also by using natural YAML syntax.

WDYT? I'm hoping I can kill two birds with one stone and address @alyssawilk ask here.

Reply:
Yeah that looks neat and a great step in the right direction! As I said offline to @htuch I think also with this Python close we are *very* close to having a Python sphinx plugin that can check examples inline. It would be really awesome to be able to add those directly in the RST files and/or link out to include things in the manfiest.

Reply:
https://github.com/envoyproxy/envoy/issues/11085